.github/workflows/nightly-release.yml

```diff
@@ -0,0 +1,78 @@
+name: NIGHTLY-RELEASE
+
+on:
+  schedule:
+    - cron: '40 23 * * *'
+env:
+  JAVA_VERSION: 17
+  SIGNING_KEY_STORE_PATH: ${{ github.workspace }}/keystore.jks
+jobs:
+  release:
+    name: Nightly release
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Check if new version is needed
+        run: |
+          if git tag --points-at HEAD | grep -q nightly-release; then
+           echo "Current nightly is already up to date"
```

Comment:
This doesn't yet stop the pipeline, does it? You could write to the env here and make other steps only run when that envvar has a specific value

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
Yes, I was looking for an option to simply cancel the action but skipping the other steps would work as well.